common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -39,6 +39,41 @@ killall_program() {
     done
 }
 
+is_wireless_iface() {
+        iwconfig "$1" > /dev/null 2>&1
+}
+
+is_bridge_iface() {
+    [ -d "/sys/class/net/$1/bridge" ] && return 0
+    return 1
+}
+
+platform_init() {
+    echo "platform init..."
+    base_mac=46:55:66:77
+    bridge_ip=192.168.250.140
+    
+    ip link add @BEEROCKS_BRIDGE_IFACE@ address "${base_mac}:00:00" type bridge
+    ip link add @BEEROCKS_BH_WIRE_IFACE@ type dummy
+    ip link add wlan0 address "${base_mac}:00:10" type dummy
+    ip link add wlan2 address "${base_mac}:00:20" type dummy
+    ip link set dev wlan0 up
+    ip link set dev wlan2 up
+    for iface in $(ifconfig | cut -d ' ' -f1| tr ':' '\n' | awk NF)
```

Comment:
Instead of adding all interfaces except for the bridge and wifi interfaces, I think it would be more appropriate to add an option to prplmesh_utils.sh that allows the caller to specify the interfaces they want to add. You'd have to add the dummy interfaces wlan0, wlan2 and WIRE_IFACE of course.

Also, ifconfig is considered deprecated. Use `ip l` instead, or for this particular case `ls /sys/class/net`. (nitpick)

Reply:
Will change this to accept parameter.